openedx/features/edly/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from model_utils.models import TimeStampedModel
+from organizations.models import Organization
+
+EDLY_SLUG_VALIDATOR = RegexValidator(r'^[0-9a-z-]*$', 'Only alphanumeric and hyphen characters are allowed.')
```

Comment:
`Only small case alphanumeric and hyphen characters are allowed.`

Reply:
Done